BranchBrowser.py

```diff
@@ -531,47 +532,73 @@ def open_config_dialog(self):
-                messagebox.showwarning("No Repositories", "No repositories found for the selected organization.")
-                return
-
-            repo_combobox['values'] = repositories
-            repo_combobox.set(self.default_repo) 
-
-        org_combobox.bind("<<ComboboxSelected>>", on_org_select)
-
         repo_label = tk.Label(config_dialog, text="Select GitHub repository:")
         repo_label.pack(anchor='w',pady=5, padx=50)
 
         repo_combobox = ttk.Combobox(config_dialog, values=[], width=30)
         repo_combobox.set(self.default_repo)  
         repo_combobox.pack(anchor='w',pady=5, padx=50)
 
+        team_label = tk.Label(config_dialog, text="Select team:")
+        team_label.pack(anchor='w', pady=5, padx=50)
+
+        team_combobox = ttk.Combobox(config_dialog, values=[], width=30)
+        team_combobox.set(self.default_team) 
+        team_combobox.pack(anchor='w', pady=5, padx=50)
+
         git_hostname_label = tk.Label(config_dialog, text="Enter GitHub hostname:")
         git_hostname_label.pack(anchor='w',pady=5, padx=50)
 
         git_hostname_entry = tk.Entry(config_dialog, width=40)
         git_hostname_entry.insert(0, 'github.com')  
         git_hostname_entry.pack(anchor='w',pady=5, padx=50)
+        
+        def load_repos_and_teams(org_name, repo_combobox, team_combobox):
+            repositories = self.github_client.get_organization_repos_names(org_name)
+            if not repositories:
+                messagebox.showwarning("No Repositories", "No repositories found for the selected organization.")
+                return
+            repo_combobox['values'] = repositories
+            repo_combobox.set(self.default_repo) 
+            
+            teams = self.github_client.get_organization_teams(org_name)
+            if not teams:
+                messagebox.showwarning("No Teams", "No teams found for the selected organization.")
+                return
+            team_combobox['values'] = teams
+            if self.default_team and self.default_team in teams:
+                team_combobox.set(self.default_team)
+            else:
+                team_combobox.set(teams[0])  # Set to the first available team if default is not found
+
+        def on_org_select(event):
+            selected_org = org_combobox.get()
+            load_repos_and_teams(selected_org, repo_combobox, team_combobox)
+
+        org_combobox.bind("<<ComboboxSelected>>", on_org_select)
+
+        # Initialize repositories and teams for the default organization on dialog open
+        load_repos_and_teams(self.default_org, repo_combobox, team_combobox)
 
         def on_save():
             new_org = org_combobox.get()
             new_repo = repo_combobox.get()
+            new_team = team_combobox.get()
             new_git_hostname = git_hostname_entry.get()
 
-            if new_org and new_repo and new_git_hostname:
+            if new_org and new_repo and new_git_hostname and new_team:

```

Comment:
Posto ovde imamo bas dosta promenljivih i sve moraju da budu true mozemo cistije to uraditi na sledeci nacin:

if all([new_org, new_repo, new_git_hostname, new_team]):